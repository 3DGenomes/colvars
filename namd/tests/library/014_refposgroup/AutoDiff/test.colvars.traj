# step         rmsd                  fa_rmsd               
           0    4.04186577623018e+00  5.95813422376982e+02 
           1    4.04610480909018e+00  5.95389519090983e+02 
           2    4.05857616367099e+00  5.94142383632901e+02 
           3    4.07999719777914e+00  5.92000280222086e+02 
           4    4.10964228294297e+00  5.89035771705703e+02 
           5    4.14781883375019e+00  5.85218116624981e+02 
           6    4.19397380897873e+00  5.80602619102127e+02 
           7    4.24810213520761e+00  5.75189786479239e+02 
           8    4.30952456066913e+00  5.69047543933087e+02 
           9    4.37717484238736e+00  5.62282515761264e+02 
          10    4.45018292735858e+00  5.54981707264142e+02 
          11    4.52690242443994e+00  5.47309757556006e+02 
          12    4.60540984853150e+00  5.39459015146850e+02 
          13    4.68332823905386e+00  5.31667176094614e+02 
          14    4.75857531393418e+00  5.24142468606582e+02 
          15    4.82728237001582e+00  5.17271762998418e+02 
          16    4.88466154736463e+00  5.11533845263537e+02 
          17    4.92366341038902e+00  5.07633658961098e+02 
          18    4.94582629752745e+00  5.05417370247255e+02 
          19    4.95958521080512e+00  5.04041478919488e+02 
          20    4.97070127538493e+00  5.02929872461507e+02 
