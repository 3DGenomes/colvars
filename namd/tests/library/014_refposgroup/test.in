colvarsTrajFrequency    1
colvarsRestartFrequency 2000

colvar {
  name r

  outputAppliedForce yes

  rmsd {
    refPositionsFile ../Common/310turn-ideal.pdb
    debugGradients yes
    atoms {
      atomNumbers  4
      rotateReference yes
      centerReference yes
      fittingGroup {
        atomNumbers  54 64 74 84 99 
      }
      refPositionsFile ../Common/310turn-ideal.pdb # ref prot positions for fitting
    }
  }
}

harmonic {
  colvars r
  centers 0.
  forceConstant 0.1
}

