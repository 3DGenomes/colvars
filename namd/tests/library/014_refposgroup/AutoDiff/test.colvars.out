colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013 in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsd
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # enableFitGradients = on [default]
colvars:       Within atom group "atoms":
colvars:       Defining atom group "refPositionsGroup".
colvars:         Initializing atom group "refPositionsGroup".
colvars:         # indexGroup = "" [default]
colvars:         # psfSegID =  [default]
colvars:         # atomsFile = ../Common/da.pdb
colvars:         # atomsCol = O
colvars:         # atomsColValue = 0 [default]
colvars:         # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:         # enableForces = on [default]
colvars:         # centerReference = off [default]
colvars:         # rotateReference = off [default]
colvars:         # enableFitGradients = on [default]
colvars:         Atom group "refPositionsGroup" defined, 104 atoms initialized: total mass = 727.821, total charge = 6.70552e-08.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = ../Common/da.pdb
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use refPositionsGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       Atom group "atoms" defined, 4 atoms initialized: total mass = 28.034, total charge = -0.1.
colvars:     The option "refPositionsGroup" (alternative group for fitting) was enabled: Jacobian derivatives of the RMSD will not be calculated.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 100
colvars:   # centers = { 10 }
colvars:   # targetCenters = { 10 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group rmsd:0/0. 4 atoms: total mass = 28.034.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
